src/ubuntu/18.04/crossdeps/Dockerfile

```diff
@@ -21,6 +21,11 @@ RUN apt-get update \
         make \
         qemu \
         qemu-user-static \
+        libbz2-dev \
```

Comment:
Please add these packages to the set in alphabetical order.

Reply:
updated.